### ISSUE TITLE
The game doesn

### ISSUE THREAD

**Issue by CalXee** [ickerio/factions#2]

**Comment by ickerio** [comment]
I totally agree. When I wrote the mod I knew it wasn't as per the classic factions rules but just wanted to get the main mechanics in place and modify later. So here's what I'll change to fix this issue

- Prevent overclaiming past faction maximum power
- All claims do not become editable when the faction does not mean the minimum power
- All claims can be overridable when the faction does not mean the minimum power

> Even jankier is that theoretically that means overriding a claim actually makes the rest of the claim once again function.

I feel like this should be intentional mechanics but please let me know if you disagree and what I could change. Thanks for the issue.
